schema-validation-lib/dev/user.clj

```diff
@@ -0,0 +1,14 @@
+(ns user
+  (:require [clojure.pprint :refer (pprint pp)]
+            [clojure.tools.namespace.repl :refer (refresh refresh-all)]
+            ;; Make Proto REPL lib properties available.
+            [proto-repl.saved-values])
+  (:use [clojure.test :only [run-all-tests]]
```

Comment:
We do not use `:use` anymore, we always use `:require`. If you look at the dev-system dev/user namespace you'll see an up-to-date example.